xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/javascript.vm

```diff
@@ -199,6 +199,11 @@ require(['jquery', 'iscroll', 'drawer'], function($, IScroll) {
       $('#tmDrawerActivator').attr('aria-expanded', 'false');
     });
   });
+    // Drawer can be closed by pressing the ESC key, irrespective of how you opened it (i.e., using keyboard or by clicking it)
+    $(document).on('keydown', function (event) {
+    if (event.keyCode === 27) { // 27 is the key code for the ESC key
+    $('.drawer-nav').closest('body').drawer('close');}
```

Comment:
The formatting does not look right.

Reply:
Also, isn't there a way to use a constant instead of an hardcoded integer value?

Reply:
Improved both. Thank you for suggesting ```.key === 'Key'```, now looks much better :)